src/Migrator.php

```diff
@@ -42,6 +42,7 @@
 use P4\MasterTheme\Migrations\M038RemoveCustomSiteIcon;
 use P4\MasterTheme\Migrations\M039EnableNewSocialSharePlatforms;
 use P4\MasterTheme\Migrations\M040ReplaceSpecialCharactersInPostsContent;
+use P4\MasterTheme\Migrations\M041SetDefaultSocialSharingOption;
```

Comment:
Hey @sagarsdeshmukh am I missing something, I see we need this file for the migration but it does not show up here.

Reply:
Hi @Osong-Michael , My bad! I forgot to add the new file in the previous commit. Could you please check now?